src/components/Timeline/Post.vue

```diff
@@ -13,11 +13,47 @@
           >
             ...
           </button>
-          <div class="dropdown-menu"
-          v-if="getUserId == post.author.id"
-          >
-            <button class="dropdown-item" type="button">Edit</button>
-            <button class="dropdown-item" type="button" @click="DeletePost(post.id)">Delete</button>
+          <div class="dropdown-menu" v-if="getUserId == post.author.id">
+            <button
+              class="dropdown-item"
+              type="button"
+              data-toggle="modal"
+              :data-target="'#updateModal_' + post.id"
+            >
+              Edit
+            </button>
+            <button class="dropdown-item" type="button">Delete</button>
```

Comment:
Where is the delete event?!

```suggestion
            <button class="dropdown-item" type="button" @click="DeletePost(post.id)">Delete</button>
```

Reply:
it exists!!